pkg/controller/controller_test.go

```diff
@@ -1511,6 +1526,42 @@ func assertAsyncOpInProgressFalse(t *testing.T, obj runtime.Object) {
 	}
 }
 
+func assertOrphanMitigationInProgressTrue(t *testing.T, obj runtime.Object) {
+	testOrphanMitigationInProgress(t, "" /* name */, fatalf, obj, true)
+}
+
+func assertOrphanMitigationInProgressFalse(t *testing.T, obj runtime.Object) {
+	testOrphanMitigationInProgress(t, "" /* name */, fatalf, obj, false)
+}
+
+func expectOrphanMitigationInProgressTrue(t *testing.T, name string, obj runtime.Object) bool {
+	return testOrphanMitigationInProgress(t, name, fatalf, obj, true)
+}
+
+func expectOrphanMitigationInProgressFalse(t *testing.T, name string, obj runtime.Object) bool {
+	return testOrphanMitigationInProgress(t, name, fatalf, obj, false)
+}
+
+func testOrphanMitigationInProgress(t *testing.T, name string, f failfFunc, obj runtime.Object, expected bool) bool {
```

Comment:
These functions should have ServiceInstance in their name to differentiate them from the ServiceInstanceCredential functions that will inevitable be coming.

Reply:
I can do that, sure.